Header header
string body
string[] actions
string[] pick_names
string[] place_names
float32[] probabilities